plugins/woocommerce/src/Admin/BlockTemplates/BlockContainerInterface.php

```diff
@@ -5,30 +5,4 @@
 /**
  * Interface for block containers.
  */
-interface BlockContainerInterface extends BlockInterface, ContainerInterface {
-	/**
-	 * Add a block container to the block container.
-	 *
-	 * @param array    $block_config The block data.
-	 * @param callable $block_creator An optional function that returns a new BlockInterface instance.
-	 *
-	 * @throws \ValueError If the block configuration is invalid.
-	 * @throws \ValueError If a block with the specified ID already exists in the template.
-	 * @throws \UnexpectedValueException If the block creator does not return an instance of BlockInterface.
-	 * @throws \UnexpectedValueException If the block container is not the parent of the block.
-	 */
-	public function &add_block_container( array $block_config, ?callable $block_creator = null ): BlockContainerInterface;
```

Comment:
This was added by Matt because I did not like the idea of all blocks allowing children.  But I've removed it since the `Block` class is now a generic catch-all block and allows children.

With the latest API, the idea is that you can create your own block by extending from `AbstractBlock` and choose whether or not to:
* Use `BlockContainerTraits` which handles inner block formatting and adds a protected `add_inner_block` method
* Implement your own public methods to add blocks, allowing as much flexibility or restriction as needed